main.go

```diff
@@ -11,21 +11,19 @@ import (
 )
 
 func main() {
-	println("Server is starting...")
+	log.Print("Server is starting...")
 
-	pokemonService, _ := service_csv.NewPokemonService(parameters.CsvPath)
+	var err error
```

Comment:
Why declare `err` here? There's no need; when using a short declaration (as in the following line) you are already doing that.